app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    flash[:notice] = "Account successfully deleted!"
+    redirect_to root_path
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @email = params[:user][:email]
+    if @user.email != @email
+      if @user.update(email: @email)
+        flash[:notice] = "Email confirmation message has been sent to your current email"
+        redirect_to root_path
```

Comment:
A kādēļ uz root path, nevis piemēram uz to pašu edit skatu vai user show skatu?

Reply:
Pirmstam, kad man rādija piemeru šajā vietā par flash messagiem , tad tur bija pie successful redirects uz root , padomaju ka tā pieņemts.